src/main/java/core/basesyntax/controller/ConsoleHandler.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax.controller;
+
+import core.basesyntax.dao.BetDao;
+import core.basesyntax.dao.BetDaoImpl;
+import core.basesyntax.model.Bet;
+import java.util.Scanner;
+
+public class ConsoleHandler {
+    BetDao betDao = new BetDaoImpl();
+
+    public void handle() {
+        Scanner scanner = new Scanner(System.in);
+        while (true) {
+            int value = 0;
+            while (value == 0) {
+                System.out.println("Enter value:");
+                try {
+                    value = Integer.parseInt(scanner.nextLine());
+                } catch (NumberFormatException e) {
+                    System.out.println("Invalid data. Enter number");
+                }
+            }
+            double risk = 0;
+            while (risk == 0) {
```

Comment:
what if user want risk to be zero?) let's use true and add exit condition inside loop

Reply:
done